modules/nw-aws-replacing-clb-with-nlb.adoc

```diff
@@ -4,18 +4,18 @@
 
 :_content-type: PROCEDURE
 [id="nw-aws-replacing-clb-with-nlb_{context}"]
-= Replacing Ingress Controller Classic Load Balancer with Network Load Balancer 
+= Replacing Ingress Controller Classic Load Balancer with Network Load Balancer
 
-You can replace an Ingress Controller that is using a Classic Load Balancer (CLB) with one that uses a Network Load Balancer (NLB) on AWS. 
+You can replace an Ingress Controller that is using a Classic Load Balancer (CLB) with one that uses a Network Load Balancer (NLB) on AWS.
 
 [WARNING]
 ====
-This procedure causes an expected outage that can last several minutes due to new DNS records propagation, new load balancers provisioning, and other factors. IP addresses and canonical names of the Ingress Controller load balancer might change after applying this procedure. 
+This procedure causes an expected outage that can last several minutes due to new DNS records propagation, new load balancers provisioning, and other factors. IP addresses and canonical names of the Ingress Controller load balancer might change after applying this procedure.
```

Comment:
You can say  "may cause an outage which might last several minutes" as generally this process is fast.  cc @Miciah

Reply:
Please note that this process changes the annotation of the service so you might end up with leaked AWS load balancer resources as per https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/service/nlb/#configuration.

Reply:
Please lmk if this change should be added to the warnings in the files `modules/nw-aws-switching-clb-with-nlb.adoc` and `modules/nw-aws-switching-nlb-with-clb.adoc` as well, since they currently include the same warnings.

Reply:
@miheer I've updated the warnings across all files, PTAL and let me know what you think. Thank you